websocket_dataframe.py

```diff
@@ -0,0 +1,66 @@
+# This is a generated file! Please edit source .ksy file and use kaitai-struct-compiler to rebuild
```

Comment:
@GreyCat, should we add all the extensions of target language files into .gitignore?

Reply:
Certainly wouldn't be opposed to it, but this was removed in the meantime.

Reply:
@KOLANICH Might make sense, albeit we currently have some UNIX shell/Ruby-based automation machinery in this repo (in `/_build`).

Want to start a PR?